ClassWork_9/Homework_7/Homework.html

```diff
@@ -0,0 +1,180 @@
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRF1avcP5Nli-Mu5mYvk1HvIhqu8QpFFumFonCHNNBwmuP9_1f4Qox_dJLTiM9ep58ZFTXEcHE&usqp=CAc",
+    name : "Women's Summer dress",
+    description : " Long Casual Elegant Dresses Party Dress ",
+    Price: "35€", 
+     shippingCost :{
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRMancKgpx0vgr9eGIYERMONz6vWT-F5XuMgWmBnrjbOD04KCmQ0KYi-lEVsTrPe6hb264ZLrhU&usqp=CAc",
+    name : "Women's Nightie",
+    description : " Short,pleasantly loose cut ,100% cotton",
+    Price: "15€", 
+     shippingCost :{
+         cost : freeShipping
+     }
+}
+
+] ;
+
+
+    
+
+
+    dressListCmp.innerHTML = `
+    <li>
+    <h5>${dressList[0].name}</h5>
+    <img  class ="dressImage" src="${dressList[0].imageUrl}" />
+    <p>${dressList[0].description}</p>
+    <p>Price:${dressList[0].Price}</p>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <p>Delivery Cost :${dressList[0].shippingCost.cost.deliveryCharge}</p>
+    <button id="firstBtn">Add to Shopping Cart </button>
+    </li>
+<li><h5>${dressList[1].name}</h5>
+    <img  class ="dressImage" src="${dressList[1].imageUrl}" />
+    <p>${dressList[1].description}</hp>
+    <p>Price:${dressList[1].Price}</p>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <p>Delivery Cost :${dressList[1].shippingCost.cost.deliveryCharge}</p>
+    <button id="secondBtn">Add to Shopping Cart </button>
+    </li>
+    <li><h5>${dressList[2].name}</h5>
+    <img  class ="dressImage" src="${dressList[2].imageUrl}" />
+    <p>${dressList[2].description}</p>
+    <p>Price:${dressList[2].Price}</p>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <p>Delivery Cost :${dressList[2].shippingCost.cost.deliveryCharge}</p>
+    <button id="thirdBtn">Add to Shopping Cart </button>
+    </li>
+    
+    <li><h5>${dressList[3].name}</h5>
+    <img  class ="dressImage" src="${dressList[3].imageUrl}" />
+    <p>${dressList[3].description}</p>
+    <p>Price:${dressList[3].Price}</p>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <span class="fa fa-star checked"></span>
+    <p>Delivery Cost :${dressList[3].shippingCost.cost.deliveryCharge}</p>
+    <button id="fourthBtn">Add to Shopping Cart </button>
+    </li>` ;
+
+    let showCart = [] ;
+
+    function addFirstItem (){
```

Comment:
could you add all dresses to the page with 1 function and an array? Just add all dresses to 1 array and execute 1 function for all of them

Reply:
Ive already did this in the latest homework ie as in the condition for checking amount